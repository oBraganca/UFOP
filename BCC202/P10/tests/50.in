50
-35 62 5 3 -8 -27 23 12 -21 14 -28 -16 20 -23 28 -6 27 1 -12 -26 -15 4 9 -29 -13 21 18 -2 -9 2 30 -24 22 -1 17 0 -25 -14 -17 -19 -31 -10 7 -3 -20 25 -4 -5 29 26 -30 6 24 -22 8 -7 10 13 -18 11 -11 15 16 19
44 51 -1 -18 -24 24 -3 19 25 8 -20 -5 -15 -14 7 23 -25 2 -7 18 11 -8 -2 -22 -21 14 16 -23 -10 -11 -9 12 -12 22 17 10 -16 13 -17 9 3 20 -13 -4 5 15 21 1 6 0 -6 -19 4
-33 44 -15 17 18 13 16 4 0 -21 -11 -8 12 10 -2 5 -1 -20 19 -19 20 11 -22 -10 14 8 -5 -4 1 -7 -16 9 -6 15 7 -3 -12 -14 21 2 -13 -17 6 -9 3 -18
65 82 19 -20 -26 -27 -19 -35 -34 -21 -31 -37 -5 -29 9 -9 -25 -14 16 3 -30 8 23 -40 -36 6 -32 -22 24 -24 -18 26 5 20 -39 30 28 -15 21 25 -23 -13 14 -28 1 27 12 2 -17 22 31 -1 -41 -6 -10 -12 11 40 36 34 -7 -3 32 37 17 33 -4 -2 -33 -38 7 10 -16 39 -11 13 29 15 35 38 4 0 -8 18
78 82 -27 39 -4 40 -28 -31 20 -16 8 14 -22 -21 29 -19 -7 -38 -40 -1 36 31 -36 18 6 32 25 35 15 34 -10 2 13 -13 -14 -20 -24 0 7 30 9 -26 -39 -25 -32 16 22 -41 38 -3 -2 21 -11 27 -5 -6 -37 17 -18 12 -17 11 -8 3 5 -29 1 23 -23 10 -15 -12 24 -30 26 33 -33 -34 -35 4 28 37 19 -9
-31 57 28 20 -24 -1 9 6 -12 -4 26 -17 -26 -25 -7 27 -21 24 -3 -6 -8 -20 -16 3 -27 -13 -2 2 0 16 -11 -9 17 12 8 23 18 22 -19 21 -5 11 -18 -22 13 10 25 14 5 4 -14 -10 7 1 19 15 -28 -15 -23
4 25 9 -6 -9 -8 0 -11 -7 12 10 -2 5 4 -12 8 -5 -1 6 2 -3 1 11 7 -4 3 -10
15 54 -25 -4 -12 -5 -14 -2 19 0 14 23 16 15 -23 -9 4 -1 -10 -16 -17 8 -19 -27 12 -22 -20 22 21 -18 -6 -8 -11 17 -26 -15 11 20 24 3 25 -13 -21 9 1 26 5 7 10 -3 18 -24 -7 6 13 2
18 72 34 -1 5 -24 -8 -3 -26 -4 -14 -28 -5 29 17 35 23 12 -13 -22 -12 -7 1 -36 27 2 -23 8 -6 -20 20 -34 3 -29 -2 13 33 -33 -10 -19 7 -16 9 19 -15 11 -35 -32 30 22 -25 15 -27 24 0 4 32 28 26 10 -17 16 31 -21 -9 6 -18 21 -31 -30 14 -11 25 18
29 62 -5 30 19 10 -30 5 -8 18 -20 15 1 21 20 -28 -25 -27 26 -21 13 -3 17 11 -11 -24 -9 -14 -12 -23 -2 12 -31 -16 -29 23 29 6 22 -4 8 -17 24 -19 4 -22 -10 25 14 3 16 -1 -7 27 -18 28 9 7 0 2 -13 -26 -15 -6
-12 67 6 10 -12 17 21 18 -29 -13 26 -19 31 20 -32 2 24 7 -16 1 -18 23 -1 11 -10 30 -23 29 -6 5 -24 27 8 -2 -30 -7 14 33 16 0 -26 -25 -4 25 -33 -15 -27 4 12 -9 13 -17 -28 32 -14 -5 9 -21 15 -3 -22 28 -31 -11 -8 19 -20 22 3
18 99 -46 0 -27 9 42 44 43 33 -19 -7 -29 -5 35 40 24 -40 12 -8 -43 34 25 -36 27 23 -15 -14 -33 -44 20 8 32 3 -1 28 -39 -10 48 47 11 37 7 45 4 -25 17 5 -48 -6 26 -31 2 6 19 39 -41 36 -26 18 41 -49 10 1 -28 22 -13 49 31 -30 -22 -12 -3 -23 -24 -35 46 15 38 -9 -45 -2 -18 -21 -32 -34 -42 -47 -38 21 -37 -16 -17 13 16 -20 29 -4 -11 14 30
-86 95 39 27 -43 34 -7 21 17 -39 15 43 -5 -19 -24 -36 -17 40 11 -14 -18 4 14 41 -41 7 13 25 28 -32 -38 0 -26 -16 -34 -27 -44 -47 24 -45 35 -15 46 3 -23 10 -42 -35 38 31 -3 -30 -31 32 30 -1 -10 -13 -40 -12 16 8 -25 -37 12 45 -20 36 42 6 33 47 -33 1 9 20 22 -11 5 37 -29 -2 23 18 -46 -6 -21 44 19 -4 -22 -9 -8 -28 2 29 26
8 38 14 15 -13 5 -7 6 2 18 -14 -15 -5 1 -11 -12 8 17 -2 -9 -6 -10 9 16 -19 7 -17 4 -8 3 0 13 -16 11 -18 -4 -3 -1 12 10
14 20 -6 1 7 0 6 5 9 -9 8 4 -1 -7 -10 -3 -8 3 -5 -2 -4 2
-22 71 -6 15 23 21 14 16 7 27 -29 6 -28 1 20 30 31 -22 29 -27 -15 -11 -23 -21 -10 9 19 32 -16 -26 -2 8 -20 4 -8 -24 -35 26 -4 25 -32 -25 33 18 -17 -33 -3 -7 10 28 22 -14 24 -18 -19 11 0 -5 -9 -30 -1 -31 17 3 13 -12 35 5 -13 34 12 -34 2
19 23 9 -1 3 11 -8 5 -10 -5 -9 -6 -3 4 0 10 -11 2 6 7 -2 1 -4 -7 8
5 67 -19 -20 -29 4 -8 20 -17 28 10 24 7 -23 -22 5 15 -21 14 8 33 -30 19 0 -9 -12 -2 27 25 13 12 -14 30 -18 -3 -16 -28 -27 -7 32 -33 -15 6 23 -5 -32 9 16 -31 29 3 -24 -26 18 -13 -6 26 22 17 31 11 -11 2 21 -25 1 -10 -1 -4
-8 34 4 -8 11 -11 10 -4 -10 1 6 -6 12 7 15 -9 -2 -13 -14 -5 13 0 9 16 -7 2 -12 14 3 5 -1 -16 -3 -17 -15 8
-24 27 13 12 -13 10 -10 6 11 -8 9 -5 0 -9 4 8 7 -3 5 -12 -4 -6 2 -1 -11 -2 3 1 -7
18 90 -18 32 -33 4 25 -20 31 19 -44 -19 42 36 43 35 -1 34 -23 38 -6 13 20 29 -29 -10 -34 39 10 -5 -38 -7 17 11 -16 33 21 -45 -32 -36 23 -2 -22 3 -13 0 -37 -11 -15 -27 14 40 27 -12 8 9 18 24 -21 -28 -3 12 -17 6 44 -39 37 -26 16 -8 5 -43 7 -9 -35 26 1 28 -24 -42 -4 -31 15 2 22 -40 -41 41 -25 -14 -30 30
66 97 47 -15 -33 32 1 -12 -43 30 18 -26 14 -29 -41 5 -9 -30 -48 -20 17 42 -8 27 2 -21 41 11 -38 -4 40 -27 -13 9 7 -17 33 39 19 25 -39 -45 4 -16 15 -28 -11 -31 -46 24 26 -36 -19 34 -2 -7 -32 36 -3 13 28 3 -42 -23 29 -35 -40 0 8 46 -10 48 20 38 -1 23 -14 44 45 21 16 31 12 22 -37 -44 -34 35 -18 -47 37 10 -22 -25 -6 -5 43 6 -24
-13 42 -17 -5 -18 -8 17 -3 -6 -1 18 -9 11 16 -19 -13 19 -4 -2 10 4 -10 7 5 -16 2 12 14 13 -7 20 15 3 -14 -15 0 -21 1 -20 6 -12 -11 8 9
-20 41 6 -11 0 10 11 4 8 -9 -3 -4 18 15 3 -20 -18 -16 -15 -6 2 -1 -19 -8 20 9 14 13 -17 19 -5 -2 -12 16 17 -14 -13 1 5 12 7 -10 -7
-5 87 -13 -21 -11 -5 32 11 -28 -2 -20 4 -25 18 -18 20 -23 28 26 -26 -32 -39 8 23 -42 29 -27 21 6 7 -16 -33 10 35 -36 3 27 34 1 -43 5 31 -19 -29 9 -22 0 13 19 -3 -30 39 25 -14 24 -1 -24 -40 -6 14 -7 12 33 -38 -4 -9 17 -10 22 42 36 -17 15 16 -37 37 2 -31 -15 -41 30 38 40 -34 -12 -8 43 41 -35
-86 99 26 5 31 -43 0 -5 -23 25 -30 39 27 45 -1 8 -41 9 -31 -9 1 -7 47 -35 17 35 -2 -36 -28 -24 13 22 -4 21 6 7 -45 -44 -19 -6 -49 18 41 23 32 -13 -42 -22 24 33 -34 37 15 -16 -10 -17 -38 38 14 16 -27 -26 -32 4 19 3 46 -33 -15 48 30 -3 43 -39 -11 -40 36 29 -21 12 -12 -25 -48 20 34 -37 44 10 -29 42 -18 -20 28 -46 49 40 11 -8 -14 -47 2
15 42 -19 -12 -13 -5 -16 -1 -17 11 0 -21 20 16 -3 6 -8 -18 -2 14 17 5 -15 -10 2 10 -9 -20 7 -7 18 -11 8 3 1 -14 19 9 12 15 4 13 -4 -6
57 63 -27 -24 24 27 0 -19 -25 9 -1 25 -3 6 2 30 13 22 -22 -11 31 21 18 -23 10 28 26 19 12 -30 1 29 -21 14 -5 5 -6 -13 -17 16 -10 -18 17 20 -16 15 -14 23 4 -2 -26 -7 -4 -12 -15 -28 -31 -29 7 -9 -20 -8 3 11 8
33 69 22 19 -20 -30 -2 -28 -29 0 18 -31 -26 -5 6 -7 2 28 -32 -16 -9 12 -12 -33 14 -17 -1 -23 3 25 -11 -24 -21 -22 -3 -18 -19 34 13 16 1 -13 -27 -15 23 -14 5 -8 32 9 -6 10 4 -4 24 15 30 21 -25 33 8 -34 31 11 20 29 17 -10 7 27 26
21 61 -11 9 -15 -18 8 -20 23 -8 -26 -23 -25 4 -2 15 -22 -24 18 -10 -9 0 25 -21 -28 22 12 -3 5 3 10 27 -13 6 17 13 -1 -4 -19 1 14 28 24 -27 30 26 -29 7 2 -30 -7 11 16 -14 -6 29 20 -17 -12 21 -16 -5 19
-44 47 -23 6 -11 -6 -13 15 12 23 -20 14 -17 -16 1 -3 18 21 -10 -14 9 -7 7 -9 5 4 11 -21 -1 -18 8 -2 -19 -22 -15 -4 -12 20 -8 2 13 19 10 -5 22 0 16 3 17
-7 23 2 9 -8 -2 0 3 -1 11 -9 1 -4 -6 7 8 10 -7 -5 5 -11 -3 -10 4 6
-28 76 32 29 -32 -24 16 -15 -20 -31 -12 24 9 -7 33 -33 -38 -8 -11 23 26 10 -10 -19 28 1 0 22 31 -9 -4 21 -5 17 -35 19 8 -1 -36 20 35 3 2 -14 7 -28 14 -34 -17 -2 -27 -22 -6 -29 36 -21 -25 -23 13 37 -3 4 25 5 6 -13 -16 -37 12 30 34 15 -26 27 18 -18 -30 11
35 43 0 -11 -8 -13 -16 -20 -5 -1 -14 -2 -7 21 -19 18 -4 17 -12 1 -9 -6 -21 15 -15 -18 9 6 13 10 16 14 4 3 19 -17 11 8 2 12 -3 5 -10 20 7
-37 91 40 36 15 -41 -12 7 -43 -31 34 35 -27 11 20 0 12 -36 43 -3 18 16 -16 -22 28 -17 -20 10 22 1 5 24 -33 2 3 33 29 -40 -11 -39 -44 -13 -18 -6 -28 -19 23 26 44 -5 -35 -15 -25 45 21 4 -37 19 -34 -26 27 -1 -21 -45 37 -42 -24 17 -38 -29 -2 -32 38 9 -4 41 32 42 -30 -7 -10 31 -8 8 25 6 14 -23 13 30 -14 -9 39
88 91 10 -45 11 -42 -40 19 -41 -28 5 -24 36 22 29 6 41 -29 -26 -15 45 -1 25 15 -2 -43 -39 -23 18 -7 -16 -19 21 34 -10 -36 0 -44 31 -20 -27 8 20 -14 17 -35 -30 -13 -18 35 27 -5 14 -38 -21 9 26 42 1 3 12 -12 33 -11 -31 -4 24 -37 40 -3 39 -33 16 4 44 7 32 2 -25 -17 38 13 -9 23 43 37 -32 -6 28 -8 -34 -22 30
-10 24 -8 -1 -10 6 8 5 -5 9 -4 4 2 1 -11 10 -12 3 -9 7 -6 -3 -2 11 -7 0
-9 21 7 -2 -10 -8 -7 0 6 10 -4 5 9 3 4 -3 1 -6 -5 -9 8 -1 2
19 92 29 -9 -5 -36 -31 9 -15 2 30 45 -8 17 -38 -37 -41 -10 28 -2 -46 -44 33 21 40 7 -6 20 22 10 6 38 -40 -45 -26 -11 11 32 43 -13 -17 5 -28 41 42 -39 25 -32 -23 -19 -22 14 -16 -4 -43 35 36 -35 -24 13 31 0 34 19 -33 -30 -14 -12 15 -20 26 -34 8 -25 -27 4 -3 -18 24 44 39 12 27 18 23 -42 16 -7 3 -1 37 1 -21 -29
10 29 -11 10 -7 -4 -13 -10 14 -12 -6 13 -9 12 -3 -2 11 6 -1 7 5 2 8 4 9 0 -14 1 -5 -8 3
-12 43 -15 12 16 -20 6 8 0 -6 -11 15 4 1 7 -3 -8 -7 17 11 13 18 9 10 -5 -12 2 19 -9 20 -2 -13 -18 -4 -10 -21 21 -17 -19 14 5 -14 -16 3 -1
12 53 -18 15 10 14 11 -20 -8 -15 2 23 22 20 -26 21 -25 24 -23 -19 -21 26 18 -11 4 -4 -16 -24 8 19 -17 0 1 -12 9 -22 -13 -10 7 -6 -5 16 3 25 6 13 -9 -2 -1 -3 12 17 5 -14 -7
-22 91 8 -14 17 -31 -3 -19 -30 -15 -25 10 3 -39 -33 -29 -8 35 31 14 24 -42 -18 9 19 -41 45 40 20 16 36 -1 22 7 41 -5 -16 -43 -13 -11 1 33 -7 26 -9 -36 42 29 30 -23 -34 38 -45 -44 6 27 21 -22 -35 28 44 15 -38 18 -40 -20 11 0 2 34 -27 5 32 4 -6 -28 -2 13 43 -26 39 37 23 -17 12 -24 25 -4 -32 -12 -10 -21 -37
-15 39 -18 18 10 -13 -4 -12 11 12 -6 -10 -9 9 5 -2 -15 14 -7 6 -3 -11 15 17 -16 -17 4 -14 -5 16 13 -19 1 8 0 -8 7 2 -1 3 19
-24 63 -19 16 -26 -25 -10 -4 1 4 -17 20 -22 -20 -3 21 13 6 -1 30 25 7 18 10 12 9 -29 -16 28 -28 -15 3 26 11 -31 -23 14 17 29 24 -18 -14 5 22 -7 23 -2 -27 -12 31 2 -21 -13 -8 -24 19 -30 15 27 8 -11 -9 -5 0 -6
-4 31 -5 -12 -1 5 -7 -11 12 15 -9 -6 4 8 -3 0 -4 -8 9 6 11 -15 -2 7 1 -13 3 -14 14 13 -10 10 2
98 100 -21 -24 -11 -33 1 -28 -4 -30 16 37 -50 21 30 14 -18 -43 -48 39 -25 -23 -26 43 13 11 12 -31 48 46 -7 -42 -35 -22 31 -17 -36 18 -40 35 33 -10 28 -8 42 -34 0 2 -15 19 15 27 -37 44 25 41 -6 45 24 -47 -39 29 3 -14 -5 7 5 6 34 -1 49 -44 -32 9 -27 -38 -49 36 -41 -12 -29 26 10 47 38 22 -20 -3 -46 32 -9 -2 4 -16 8 -13 17 -45 23 40 20 -19
26 57 11 -22 8 -5 2 1 -9 -19 19 -8 -16 22 0 25 3 -4 6 -27 -23 9 -18 -13 20 27 -6 7 -25 -24 21 4 -20 23 -10 -1 -15 16 -21 17 18 10 -26 -14 -17 -7 12 -11 14 -28 24 5 -3 -12 26 13 28 -2 15
55 87 31 -33 -9 8 -8 28 -42 -29 4 -23 37 -12 17 32 24 -43 25 27 19 -21 -3 -40 -19 43 22 34 -35 40 30 -15 -27 -34 21 -2 -31 -20 35 15 -26 6 41 -30 -24 -16 12 -39 -41 -13 -36 -5 -1 2 -22 33 -28 -4 16 11 0 13 -14 20 -32 7 -17 -6 23 9 39 3 -38 -11 -18 38 1 26 -10 -37 10 5 -7 36 -25 14 29 42 18
64 87 30 -1 -6 0 19 -30 15 -7 13 3 32 -2 8 -13 12 -9 18 -37 29 -41 17 -8 -34 27 -3 31 -32 -40 -10 10 -11 36 33 -5 -24 -18 -25 21 -17 -16 4 23 1 9 -33 11 16 24 6 -19 -20 -26 -15 25 -43 40 42 -27 2 22 37 -39 -14 7 -36 -28 14 -21 41 -29 38 -42 28 43 -12 34 35 -35 -22 20 -38 5 26 -4 39 -31 -23
