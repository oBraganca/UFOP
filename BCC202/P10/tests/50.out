62 5
51 6
44 5
82 7
82 7
57 5
12 4
12 6
72 6
35 5
3 6
58 4
95 6
15 6
20 5
16 6
23 5
14 6
2 4
27 4
55 7
97 7
14 6
14 5
4 6
99 6
38 3
63 6
58 6
58 5
47 5
16 5
44 5
43 6
55 6
91 7
3 3
18 3
62 5
2 5
24 6
49 6
56 6
15 3
53 3
15 3
100 7
53 6
87 7
87 7
